### PR TITLE
Release v0.4.1

### PR DESCRIPTION
**Pull request by gronke**
- generate docs from setuptools `python3.6 setup.py build_sphinx`
- load version information from central `VERSION` file included in the build

Will require a manual update of the build scrips in the gh-pages branch.

### CODE REVIEW

File: ioc/VERSION [hunk 1]
```diff
@@ -0,0 +1 @@
+0.4.1 2019/01/05
```
Comment (igalic) [inline ioc/VERSION]: should maybe be auto-generated on release

File: docs/conf.py [hunk 2]
```diff
@@ -15,18 +15,32 @@
...
 # import sys
 # sys.path.insert(0, '/usr/local/src/libioc/iocage')
-
+import sphinx.apidoc
+import os.path
+
+def setup(app):
+    sphinx.ext.apidoc.main([
+        '--separate',
+        '-H', 'FOOOO',
```
Comment (igalic) [inline docs/conf.py]: what's FOOOO?

File: ioc/__init__.py [hunk 3]
```diff
@@ -24,62 +24,70 @@
 # POSSIBILITY OF SUCH DAMAGE.
 """Libioc Python module ioc."""
```
Comment (igalic) [inline ioc/__init__.py]: what does that mean?

File: docs/conf.py [hunk 4]
```diff
@@ -15,18 +15,32 @@
...
+        'ioc',
+        'tests'
+    ])
 
 # -- Project information -----------------------------------------------------
 
 project = 'libioc'
-copyright = '2018, ioc Contributors'
+copyright = '2019, ioc Contributors'
 author = 'ioc Contributors'
```
Comment (igalic) [inline docs/conf.py]: it's funny that the docs are © ioc Contributors, but all the code is © Stefan Grönke

Comment (gronke) [reply docs/conf.py]: Changed it to automatically obtain the copyright info from `LICENSE.txt`. This automation step eases keeping all information current.

File: ioc/VERSION (hunk 1 above)
Comment (gronke) [reply ioc/VERSION]: Cutting releases is a manual process for now.

**Pull request merged**